from_cpython/Python/mystrtoul.c

```diff
@@ -92,7 +92,7 @@ static int digitlimit[] = {
 **              exceptions - we don't check for them.
 */
 unsigned long
-PyOS_strtoul(register char *str, char **ptr, int base)
+PyOS_strtoul(register const char *str, char **ptr, int base)
```

Comment:
hmm I feel like these two arguments should have the same const-ness -- changing the argument to be const so that we can cast the constness away seems like it defeats the purpose :P  my preference would be to have both arguments be const, or I guess we could have them both be non-const and make the caller be ok with casting away the constness?